envo.owl

```diff
@@ -143489,6 +143489,696 @@ Classes for population already  exist in IDO (&apos;organism population&apos;, I
     
 
 
+    <!-- http://purl.obolibrary.org/obo/envo.owl#OWLClass_03510000 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/envo.owl#OWLClass_03510000">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/ENVO_00002008"/>
+        <obo:IAO_0000115>A light- to dark-colored granular solid mixture that is generated when timber is processed, such as when it is chipped, sawed, turned, drilled or sanded. The chemical composition of wood dust depends on the species of tree and consists mainly of cellulose, polyoses and lignin, with a large and variable number of substances with lower relative molecular mass. Wood dust is not typically produced for specific uses. However, it is used commercially for wood composts. The primary routes of potential human exposure to wood dust are inhalation and dermal contact. It is known to be a human carcinogen.</obo:IAO_0000115>
+        <dc:creator>https://orcid.org/0000-0002-7463-6306</dc:creator>
+        <oboInOwl:hasDbXref>NCIT_ C44463</oboInOwl:hasDbXref>
```

Comment:
Please add full IRIs for xrefs where possible. CURIE form (NCIT:C44463) is acceptable, but full IRI form is much preferred.

Reply:
fixed